### PR TITLE
Improve NHC docs

### PR DESCRIPTION
**Pull request by banool**
I was answering questions in Discord and noticed that these docs were quite of date compared to NHC as it is today. This PR fixes that.

### CODE REVIEW

File: developer-docs-site/docs/nodes/measure/node-health-checker.md [hunk 1]
```diff
@@ -16,24 +16,22 @@ If you are a node operator, use the NHC service to check if your node is running
...
-:::
-
 This document describes how to run NHC locally when you are operating a node.
 
 ## Quickstart
 
 Before you get into the details of how NHC works, you can run the below steps to start the NHC service and send it a request. This quickstart uses a baseline configuration for a devnet fullnode, i.e., it will evaluate your node against a devnet fullnode that is configured with the baseline configuration YAML.
 
-**Important**: If your local node is not a devnet fullnode, e.g. it is a single testnet node for AIT3 registration, you must use a different configuration file. See [the configuration examples in aptos-core](https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/node-checker/configuration_examples) for other such example configs.
+**Important**: If your local node is not a devnet fullnode, you must use a different baseline config. See [the configuration examples in aptos-core](https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/node-checker/configuration_examples) for other such example configs.
```
Comment (clay-aptos) [inline developer-docs-site/docs/nodes/measure/node-health-checker.md:25]: This:
https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/node-checker/configuration_examples
Says, "These instructions are for the Aptos Labs team."





[10:28](https://aptos-org.slack.com/archives/D0464TERLAY/p1677781725306029)
That alright?

**Review by clay-aptos — COMMENTED** [review]
Thanks, Daniel. See my one comment regarding a potential update to:
https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/node-checker/configuration_examples

I am adding Greg for review too.

**Comment by clay-aptos** [comment]
See also:
https://github.com/aptos-labs/aptos-core/pull/6871

**Review by davidiw — APPROVED** [review]

File: developer-docs-site/docs/nodes/measure/node-health-checker.md (hunk 1 above)
Comment (davidiw) [reply developer-docs-site/docs/nodes/measure/node-health-checker.md:25]: I think we should consider moving that to internal documentation.

**Pull request merged**